content/troubleshooting/common-issues.md

```diff
@@ -28,6 +27,25 @@ ssh://XXXXXXXX@git-codecommit.us-west-2.amazonaws.com/v1/repos/my_repo.git
 Please note that the ***XXXXXXXXX** refers to your **SSH-Key-ID**
 
 
+### Cannot access the repository. Request is unauthorized (401)
+
+###### Description
+
+After switching your GitHub account to organization or making significant changes, you might encounter the following error when trying to select an app to build:
+
+```
+"Cannot access the repository. Request is unauthorized (401). Please check your credentials to access https://api.github.com/repos/myrepo/myrepo-flutter/branches?per_page=100."
+```
+
+###### Solution
+
+The issue may be related to the integration token requiring a refresh. To solve this, try disconnecting the GitHub integration and then connecting it again.
```

Comment:
The following suggestions can help resolve the issue:

1. Make sure that the access credentials e.g. SSH key pairs were added correctly
2. In general, access to repository needs to be updated. More information [here](https://docs.codemagic.io/getting-started/adding-apps/#modifying-access)
3. Make sure [IP addresses](https://docs.codemagic.io/getting-started/adding-apps/#firewall-configuration-for-privately-hosted-repositories) are whitelisted
4. Refresh the OAuth integration by going to Team settings .........